app/src/main/java/com/sdp/movemeet/Activity/ActivityDescriptionActivity.java

```diff
@@ -351,28 +303,11 @@ public void changeActivityPicture(View view) {
     protected void onActivityResult(int requestCode, int resultCode, @androidx.annotation.Nullable Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
         if (requestCode == 1000) {
-            if(resultCode == android.app.Activity.RESULT_OK) {
+            if (resultCode == android.app.Activity.RESULT_OK) {
                 Uri imageUri = data.getData();
                 progressBar.setVisibility(View.VISIBLE);
                 FirebaseInteraction.uploadImageToFirebase(storageReference, imagePath, imageUri, activityImage, progressBar);
             }
         }
     }
-
-    /*public void goToHome(View view){
-        Intent i = new Intent(ActivityDescriptionActivity.this, HomeScreenActivity.class);
-        i.putExtra(EXTRA_ACTIVITY_ID, "1");
-        i.putExtra(EXTRA_ORGANISATOR_ID, "1");
-        i.putExtra(EXTRA_ADDRESS, "1");
-        i.putExtra(String.valueOf(EXTRA_DURATION), 1.0);
-        i.putExtra(EXTRA_DESCRIPTION, "1");
-        i.putExtra(EXTRA_TITLE, "1");
-        i.putExtra(String.valueOf(EXTRA_DATE), "1");
-        i.putExtra(String.valueOf(EXTRA_LATITUDE), 1.0);
-        i.putExtra(String.valueOf(EXTRA_LONGITUDE), 1.0);
-        i.putExtra(String.valueOf(EXTRA_NUMBER_PARTICIPANT), 1.0);
-        i.putExtra(String.valueOf(EXTRA_SPORT), "1.0");
-
-        startActivity(i);
-    }*/
```

Comment:
Yes, I was hesitating to remove this part as well since it was commented since a while, I will do the same!

Reply:
Yes, it's was for an old task, so we don't need it anymore